packages/connectivity/src/scp-cf/destination/destination-from-env.spec.ts

```diff
@@ -251,17 +251,6 @@ describe('registerDestination', () => {
     ).resolves.toMatchObject(mockDestinationFromEnv);
   });
 
-  it('should throw an exception if a property of the destination is missing', () => {
```

Comment:
Why?

Reply:
I think its impossible to test for compile errors, that's why I removed it instead (since the test can't run anymore)